rest/model/task.go

```diff
@@ -191,7 +191,7 @@ type ApiTaskEndDetail struct {
 func (at *ApiTaskEndDetail) BuildFromService(t apimodels.TaskEndDetail) error {
 	at.Status = utility.ToStringPtr(t.Status)
 	at.Type = utility.ToStringPtr(t.Type)
-	// Update this condition DEVPROD-9694
+	// Update this condition in DEVPROD-9694.
```

Comment:
nit. The typical pattern is to use the verbiage TODO followed by the ticket number and then the change that needs to happen. This makes it easy to find outstanding TODO's in the codebase
ex:
```go
// TODO DEVPROD-9694: Stop storing failing command in Description 
```